README.md

```diff
@@ -1,18 +1,40 @@
 # Kotlin Android Architecture Library - Kaal - by eMan
 
+[![Latest version](https://img.shields.io/github/v/release/eManPrague/kaal)](https://github.com/eManPrague/kaal/releases/tag/v0.8.0)
+
 [![Slack channel](https://img.shields.io/badge/Chat-Slack-blue.svg)](https://kotlinlang.slack.com/messages/kaal/)
 
 ### Usage
 
-All artifacts are available and distributed by using the `jcenter()`
+:warning: The artifacts was moved from jCenter.
 
-You should you artifact which you need. E.g. a domain artifact you will use in your domain layer, 
-but you can use it also in data and infrastructure, because you need e.g. instance to the `Result` class.
+All artifacts are available and distributed by using the [eMan Nexus](https://nexus.eman.cz/service/rest/repository/browse/maven-public/) repository.
+Add the repository to project `build.gradle.kts` (`build.gradle`) file.
+
+```kotlin
+allprojects {
+
+    repositories {
+        ...
+        maven(url = "https://nexus.eman.cz/repository/maven-public")
+    }
+}
+```
 
+```groovy
+allprojects {
 
+    repositories {
+        ...
+        maven { url 'https://nexus.eman.cz/repository/maven-public' }
+    }
+}
+```
+
+You should you artifact which you need. E.g. a domain artifact you will use in your domain layer,
```

Comment:
`You should you`? Nemyslel jsi `You should use`?

Reply:
Nevím, nepsal jsem to, ale mrknu na to :)